.env.example

```diff
@@ -0,0 +1 @@
+BLOCK_SECURITY_KEY=
```

Comment:
can't find any usage for this key @danvincent11 

Migrate and use  ETHEREUM_RPC_URL for env instead for ethRPC in `publicRuntimeConfig` .. Remember to not include the key